config/kubernetes-sigs/sig-api-machinery/teams.yaml

```diff
@@ -76,18 +85,6 @@ teams:
     - pwittrock
     - seans3
     privacy: closed
-  kubebuilder-declarative-pattern-admins:
-    description: "admin access to kubebuilder-declarative-pattern"
-    members:
-    # - johnsonj # TODO: uncomment once johnsonj is a member, ref: https://github.com/kubernetes/org/issues/418#issuecomment-465357261
```

Comment:
I didn't add this back because johnsonj is still not a member. I would prefer to add them when they become a member instead of having a TODO linger here.

Reply:
Now I'm thinking about it, I agree that it isn't useful to include this one again. When johnsonj wants to become a member we can easily add him again.

Reply:
Membership incoming: https://github.com/kubernetes/org/issues/510

Reply:
Awesome, approved #510 and added him. :+1: